languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -56241,6 +56241,87 @@ though, and has since been slightly modified)-->
             </rule>
         </rulegroup>
         <rulegroup id="BARBARISMES_COURANTS" name="Barbarismes courants divers" default="temp_off">
+            <rule id="CONFORME_1" name="conforme avec->à">
+                <pattern>
+                    <marker>
+                        <token regexp="yes">conformes?</token>
+                        <token>avec</token>
+                    </marker>
+                </pattern>
+                <message>'\1 \2' est une expression fautive. </message>
```

Comment:
Excessive space at the end of the message

Reply:
done